tiingo/api.py

```diff
@@ -19,15 +18,15 @@
     MissingRequiredArgumentError,
 )
 
+from tiingo.__version__ import __version__ as VERSION
```

Comment:
I wonder why we couldn't have used this syntax to begin with 😅 . Assuming all unit tests pass, this looks good to me. Thank you!